geometry/src/units.rs

```diff
@@ -94,6 +94,10 @@ pub fn ratio_ (x: Ratio) -> f32 { x.get::<uom::si::ratio::ratio>() }
 pub fn radian_(x: Angle) -> f32 { x.get::<uom::si::angle::radian>() }
 pub fn turn_  (x: Angle) -> f32 { x.get::<uom::si::angle::revolution>() }
 
+pub fn mm_ps (x: f32) -> Velocity { m_s (x * 1e+9) }
+pub fn mm_ps_(x: Velocity) -> f32 { m_s_(x * 1e-9) }
```

Comment:
I'm not a huge fan of the hard wired `1e9`. Maybe it'd be more robust to have a function or constant that does the conversion. Without too much thinking I found:

`fn mpsec() -> f32 {
  Velocity::new::<meter_per_second>(1.0).value
}
pub fn mm_ps (x: f32) -> Velocity { m_s (x / mpsec()) }
pub fn mm_ps_(x: Velocity) -> f32 { m_s_(x * mpsec()) }`

But there's maybe something a bit less brute force.

Reply:
Oh, I do like this.

(I've edited your comment: it was very difficult to read with single backticks.)